packages/code-display/src/CodeDisplay.tsx

```diff
@@ -36,20 +37,28 @@ import { getPrismLanguage, getOverridenTheme } from './utils'
 // require("prismjs/components/prism-csharp")
 // require("prismjs/components/prism-swift")
 
-interface PrismEditorProps {
+interface CodeDisplayProps {
   /** SDK programming language */
   language?: string
   /** Code blob to be highlighted */
   code: string
   /** Pattern to be search (if applicable) */
   pattern?: string
+  /** onChange event handler, for Editor functionality */
+  onChange?: (text: string) => void
+  /** whether or not to render transparent background */
+  transparent?: boolean
+  height?: string
```

Comment:
Add a comment perhaps describing what happens if no height is specified? I don't see this prop being used anywhere